src/vsphere_cpi/spec/integration/ip_conflict_detection_spec.rb

```diff
@@ -56,4 +57,115 @@
       end
     end
   end
+
+  context 'when testing with fully qualified network names' do
+    context 'when a VM already exists on network' do
+      context 'when the VM exists on a different network with same unqualified name' do
+
+        let(:network_spec_1) do
+          {
+            'static' => {
+              'ip' => fixed_ip_address,
+              'netmask' => '255.255.254.0',
+              'cloud_properties' => {'name' => vlan_1},
+              'default' => ['dns', 'gateway'],
+              'dns' => ['169.254.1.2'],
+              'gateway' => '169.254.1.3'
+            }
+          }
+        end
+
+        let(:network_spec_2) do
+          {
+            'static' => {
+              'ip' => fixed_ip_address,
+              'netmask' => '255.255.254.0',
+              'cloud_properties' => {'name' => vlan_2},
```

Comment:
context says same unqualified name, looks like this test has 2 different names. If thats the case please update the context statement accordingly

Reply:
vlan1 and vlan2 are fetched from vcpi-nimbus/recipe/environment.sh, they have been set to same unqualified names:

export BOSH_VSPHERE_CPI_FOLDER_PORTGROUP_ONE='vcpi-network-folder/vcpi-switch-5/vcpi-same-uq-1'
export BOSH_VSPHERE_CPI_FOLDER_PORTGROUOP_TWO='vcpi-switch-4/vcpi-same-uq-1'